src/main/java/org/zendesk/client/v2/model/CustomFieldValue.java

```diff
@@ -32,11 +36,18 @@ public void setId(Long id) {
         this.id = id;
     }
 
-    public String getValue() {
+    public String[] getValue() {
         return value;
     }
 
-    public void setValue(String value) {
+    public void setValue(String[] value) {
         this.value = value;
     }
+
+    @Override
+    public String toString() {
+        return "CustomFieldValue{" + "id=" + id +
+                ", value=" + (value == null ? "null" : Arrays.asList(value).toString()) +
```

Comment:
can be simplier: showing empty array even if the value is `null` is not really a big deal imo. Also, `toString` are usually only used for logger..